R/return_linter.R

```diff
@@ -73,16 +74,25 @@ return_linter <- function(
     return_style = c("implicit", "explicit"),
     allow_implicit_else = TRUE,
     return_functions = NULL,
-    except = NULL) {
+    except = NULL,
+    except_regex = NULL) {
   return_style <- match.arg(return_style)
 
-  if (!allow_implicit_else || return_style == "explicit") {
-    except_xpath <- glue("parent::expr[not(
+  check_except <- !allow_implicit_else || return_style == "explicit"
+  # We defer building the XPath strings in this case since we can't build the
```

Comment:
Add a TODO referencing the XPath 2.0 issue?

Reply:
Great idea, but I think there are a bunch of places with the same issue... #2453